README.md

```diff
@@ -5,19 +5,73 @@
+* A chef that makes the food
 
-Introduce the concept of Model View Controller from a rails perspective.
+* A waiter that takes the order and brings out the food
 
-Diagram a request flow, from browser, to http request, to router, to controller, to model, to view, to controller, to http response, to browser.
+* A table where the food is served to the customer
 
-Describe the responsibilities of the different kinds of code you write, what goes where, data code, model logic, controller rendering/redirect logic, view / presentation logic, routing, even actionview helpers. A quick example would be helpful. 
+Using this as an analogy, we can assign the following roles to each of our restaurant components:
 
+* Model - The model is the chef, it should manage the critical aspects of the application. One of my favorite tasks in a Rails application is working with the model files. This is where you can be very expressive with the custom algorithms that you want to utilize and you also have direct access to the specific database record. The logic of your application should mainly reside in the model files.
+
+* Controller - The controller is the waiter. In the same way that the waiter takes the order from the customer to the chef and then brings the food to the table, the controller transmits data requests from the user to the model, and then delivers data that is rendered in the view to the user.
+
+* View - The view is the table. A table shouldn't do anything besides sit there and hold the food when it is delivered. In like manner the views should not contain any programming logic, they should simply render what the controller sends it. One of the top issues I discover when I review a Rails application that has a large number of bugs is that the developer integrated too much logic directly into the view.
+
+## Routing, File Naming Conventions, and Data Flow
+
+Rails was created with the concept of convention over configuration and this holds true for how the MVC structure was setup. View files correspond directly to controller files, which speak directly with models. As an example, imagine that you have a blog that has a database table called ```posts```. You will have the following set of files:
+
+* A ```post.rb``` model file that will contain: validations, database relationships, callbacks, and any custom logic for posts.
+
+* A ```posts_controller.rb``` file that will have methods that will manage data flow for the Post behavior, this will include the full set of CRUD features, the standard methods are: index, new, create, show, edit, update, and destroy.
+
+* A ```views/``` directory that will contain a corresponding view for each of the pages that the end user will access. For a CRUD based model, a few of the standard views would include: an index view to show all records, a show page that shows a specific record, and then new and edit pages that both render a form.
+
+## Request Flow
+![MVC Request Flow](http://reif.io/lib/flatiron/mvc-flow.jpg)
```

Comment:
All valid, I've built a new diagram from scratch and I'll add it to the new PR, you can see it here: ![MVC](https://s3.amazonaws.com/flatiron-bucket/readme-lessons/mvc_flow_updated.png)